lib/src/visitor/async_evaluate.dart

```diff
@@ -1306,13 +1285,20 @@ class _EvaluateVisitor
         var oldParent = _parent;
         var oldEndOfImports = _endOfImports;
         var oldOutOfOrderImports = _outOfOrderImports;
+        var oldConfiguration = _configuration;
         _importer = importer;
         _stylesheet = stylesheet;
         _root = ModifiableCssStylesheet(stylesheet.span);
         _parent = _root;
         _endOfImports = 0;
         _outOfOrderImports = null;
 
+        // This configuration is only used if it passes through a `@forward`
+        // rule, so we avoid creating unnecessary ones for performance reasons.
+        if (stylesheet.forwards.isNotEmpty) {
```

Comment:
Before you submit, try removing this condition and *always* creating the environment, and run the result against sass-spec. It shouldn't change the behavior, so if it does you'll have found a bug.